pkg/receiver.go

```diff
@@ -225,7 +225,7 @@ func (v *WebRTCVideoReceiver) receiveRTP() {
 
 		v.buffer.Push(pkt)
 
-		if v.feedback == webrtc.TypeRTCPFBTransportCC {
```

Comment:
shouldn't this be set based on whether client supports it?

Reply:
Yes.
Setting some params by JS-SDK is better. We can use it by settting on sfu config file recently.
I think we can merge first and do a better api setting on sdk when JS-SDK is ready